### ISSUE TITLE
yaml: `axis: id: <int>`; type safety is preventing the use EPICS Macro

### ISSUE THREAD

**Issue by kivel** [paulscherrerinstitute/ecmccfg#163]
```yaml
axis:
  id: ${AXIS_ID=1}
```
is raising and error

```
SyntaxError: 
axis:
- id:
  - must be of integer type
```

**Comment by kivel** [comment]
all templates that use `{{ axis.id|int }}` will render to `0` when a string is provided.
Those need to changed to `{{ axis.id }}`, no need to cast, as `cerberus` takes care of the types anyway.

**Comment by kivel** [comment]
fixed 7df87925c4d4d8f539dec3391338fb89c6b77022
